commitizen/cli.py

```diff
@@ -78,7 +77,6 @@ def load_cfg():
     for cfg in configs:
         if not os.path.exists(config_file) and os.path.exists(cfg):
             config_file = cfg
-            break
```

Comment:
This `break` is not needed. If we remove it, then the app will pick up the latest. 
The idea is to pick up the first one and continue.

Reply:
But there is another break after `settings.update(dict(config.items("commitizen")))`, and it ensures to pick up the first one and continue.